lib/OpenQA/WebAPI/Plugin/Gru.pm

```diff
@@ -82,6 +92,14 @@ sub is_task_active {
     return $self->count_jobs($task, ['active']) > 0;
 }
 
+# checks if there are worker registered
+sub has_workers {
+    my ($self) = @_;
+
+    my $worker_count = $self->app->minion->backend->list_workers(0, 1)->{total};
+    return defined($worker_count) && $worker_count > 0;
+}
```

Comment:
No need to be so verbose, just typecast to boolean.
```perl
sub has_workers { !!$self->app->minion->backend->list_workers(0, 1)->{total} }
```

Reply:
changed to boolean typecast